bigflow/_version.py

```diff
@@ -1,2 +1 @@
-__version__ = '1.4.2'
-
+__version__ = '1.4.3dev1'
```

Comment:
As we removed two features here (logs command and dataproc support), I think it should be 1.5.0. 

BTW Did we announce earlier that those features are deprecated and will be removed?

Reply:
True, I will change it to 1.5.0dev1.

Yes, these features have been deprecated since Feb 2022 (although there was no slack or email communication). 

We know that inside the company there isn't a single person using Dataproc support.

There are some cases of "using" (people add the logging extension, but have no idea how it can be used) the log extras.

I opt for deleting the logging stuff anyway. Removing it from a bigflow project after bumping it to 1.5.0 comes down to removing a single property (log_config) from workflows and removing the log extras from bigflow in requirements.in.

Because the log extras is easily detectable through github search, I can just find these projects and create pull requests removing the log extras.

wdyt?

Reply:
Sounds good, do it as you wrote.

Reply:
ok